extension/content.js

```diff
@@ -194,6 +215,8 @@ document.addEventListener('DOMContentLoaded', () => {
 	}
 
 	if (isRepo) {
+		const isRepoTree = window.location.href.split('/').length === 5 || window.location.href.split('/')[5] === 'tree';
```

Comment:
Is ```isRepoRoot``` ok to check? As it will only only allow to edit the main readme. There are readmes in sub-directories also.